src/main/java/pizzashop/controller/KitchenGUIController.java

```diff
@@ -38,6 +41,7 @@ public void run() {
                   } catch (InterruptedException ex) {
                     break;
                 }
+               b=j<Integer.MIN_VALUE;
```

Comment:
cred ca din cauza statement-ului astuia, o sa se opreasca dupa prima iteratie

Reply:
se duce pana in Integer.MAX_VALUE+1, adica cand depaseste limita de integer pozitive si trece pe valori negative

Reply:
Pornesti cu j=0, b=true. Faci j++ in while, j devine 1. Ajunge la b =  j<Integer.MIN_VALUE: 1 < -2 miliarde si whatever? false => b=false, se opreste. Ruleaza si va fi asa.